packages/website/.env

```diff
@@ -1 +1 @@
-../../../../secret/iolab-env
\ No newline at end of file
+DATABASE_URL="file:./dev.db"
```

Comment:
It should be a file, not a redirection link. Delete it and recreate the file with the same content.

Reply:
I create the file ,but it cant work :<

  25    return json(data)
  26 }
  27 const getData = async (articleBoxId = 1) => {
→ 28    const data: IArticleBox = await db.articleBox.findUnique(
error: Environment variable not found: DATABASE_URL.
  -->  schema.prisma:10
   |
 9 |   provider = "sqlite"
10 |   url      = env("DATABASE_URL")
   |

Reply:
*DO NOT RESOLVE CONVERSATION* by yourself, if it is not solved/fixed definitely.

Check if `https://www.npmjs.com/package/dotenv` helps.

Reply:
like this DATABASE_URL="file:../../../../secret/iolab-env.db" ? :<